README.md

```diff
@@ -1,14 +1,14 @@
 <img src="logo-gopherblue.png" alt="" width="448" height="221">
 
-# The Go Cloud Project
+# The Go Cloud Development Kit (Go CDK)
 
 _Write once, run on any cloud ☁️_
 
 [![Build Status](https://travis-ci.com/google/go-cloud.svg?branch=master)][travis]
 [![godoc](https://godoc.org/github.com/google/go-cloud?status.svg)][godoc]
 [![Coverage Status](https://coveralls.io/repos/github/google/go-cloud/badge.svg?branch=master)](https://coveralls.io/github/google/go-cloud?branch=master)
 
-The Go Cloud Project is an initiative that will allow application developers to
+The Go Cloud Development Kit (Go CDK) will allow application developers to
```

Comment:
Drop the "will". It's cleaner (and matches verb tense with the next sentence).

```suggestion
The Go Cloud Development Kit (Go CDK) allows application developers to
```

Reply:
Done.